app/lib/contentdm/log.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+##
+# This class is initilized wih a string and level (info, error) and then outputs
+# the message to the console and a file. The levels are listed here:
+# https://ruby-doc.org/stdlib-2.5.0/libdoc/logger/rdoc/Logger.html
+module Contentdm
+  class Log
+    ##
+    # @param [String] message level
+    def initialize(message, level, file = File.open(Rails.root.join('log', 'contentdm_import.log'), 'w'))
```

Comment:
You are opening a file here without closing it.  I think you need to call the `close` method on the loggers at the end of the method: 

```
std_out_logger.close
file_logger.close
```

Reply:
@val99erie I don't think the `std_out_logger` is necessary, but I did close the `file_logger` since I'm opening a `File`.